pkg/objstore/s3/s3.go

```diff
@@ -87,6 +89,11 @@ func parseConfig(conf []byte) (Config, error) {
 		return Config{}, err
 	}
 
+	// here for testing
```

Comment:
This looks wrong as @povilasv mentioned in https://github.com/thanos-io/thanos/pull/1392/files#r312902398